src/components/EnterpriseSubsidiesContext/data/tests/hooks.test.js

```diff
@@ -57,30 +79,87 @@ describe('useEnterpriseOffers', () => {
     });
   });
 
-  it.each([0, 2])('should set canManageLearnerCredit to false if enterprise does not have exactly 1 offer', async (
-    offersCount,
-  ) => {
-    const mockOffers = [...Array(offersCount)].map((_, index) => ({
-      uuid: `offer-${index}`,
+  it('should fetch enterprise offers for the enterprise when data available in enterprise-subsidy', async () => {
+    const mockOffers = [
+      {
+        id: 'offer-id',
+        name: 'offer-name',
+        start: '2021-05-15T19:56:09Z',
+        end: '2100-05-15T19:56:09Z',
+        isCurrent: true,
+      },
+    ];
+    const mockSubsidyServiceResponse = [{
+      uuid: 'offer-id',
+      title: 'offer-name',
+      active_datetime: '2021-05-15T19:56:09Z',
+      expiration_datetime: '2100-05-15T19:56:09Z',
+    }];
+    SubsidyApiService.getSubsidyByCustomerUUID.mockResolvedValueOnce({
+      data: {
+        results: mockSubsidyServiceResponse,
+      },
+    });
+
+    const { result, waitForNextUpdate } = renderHook(() => useEnterpriseOffers({
+      enablePortalLearnerCreditManagementScreen: true,
+      enterpriseId: TEST_ENTERPRISE_UUID,
     }));
 
+    await waitForNextUpdate();
+
+    expect(SubsidyApiService.getSubsidyByCustomerUUID).toHaveBeenCalledWith(TEST_ENTERPRISE_UUID);
+    expect(result.current).toEqual({
+      offers: mockOffers,
+      isLoading: false,
+      canManageLearnerCredit: true,
+    });
+  });
+
+  it('should set canManageLearnerCredit to false if enterprise does not have exactly 1 offer', async () => {
```

Comment:
nit: perhaps update test case name to mention both offer or enterprise subsidy, given either of them would make `canManageLearnerCredit` truthy?